script.js

```diff
@@ -1,4 +1,79 @@
+//include current gamemode
+var currentGameMode = 'insert username to begin';
+//include option for username
+var userName = '';
+// track number of games user wins
+var numUserWin = 0;
+// track number of games user lost
+var numUserLost = 0;
+//track number of games user draws
+var numUserDraw = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // first game mode is when username is keyed in
+  if (currentGameMode == "insert username to begin") {
+    userName = input;
+    currentGameMode = "commence scissors paper stone game";
+    myOutputValue = "hello " + input + " you can now begin the game.";
+// second game mode is when username is not keyed in 
+  } else if (!input) {
+    currentGameMode = "insert username to begin";
+    userName = !input;
+    myOutputValue = "error please insert username to begin";
+  }
   return myOutputValue;
 };
```

Comment:
There is another main() from line 54 onwards below. There cannot be 2 functions with the same name. When main() is invoked, computer will not know which main() to use.
The contents of main() from line 54 below should be included in this main().